### ISSUE TITLE
end-to-end fine-tuning

### ISSUE THREAD

**Issue by ciwei123** [lidq92/MDTVSFA#7]
@lidq92 Thanks for your reply. When I train the network end to end ,that means I will train the Resnet-50,overfifting will occur,SROCC is 0.1.But if I just only train the regression net, the SROCC is 0.6. What should I do?Thank you very much!!

**Comment by lidq92** [comment]
@ciwei123 Please read the paper and follow the `README`. I did not get what you said both in the title and the comment.

**Comment by lidq92** [comment]
@ciwei123 Please change your issue's title to the correct one. I tried to understand your question, and if my understanding is correct, then one suggestion for end-to-end fine-tuning is using a smaller learning rate for the pretrained ResNet-50's weights.

**Comment by ciwei123** [comment]
@lidq92 Thanks for your reply. I will try the method you said. I am sorry for my carelessness about the title.Thank you very much.

**Comment by lidq92** [comment]
@ciwei123 Usually 1/10*lr for the ResNet-50 backbone's weights. See [lidq92/LinearityIQA](https://github.com/lidq92/LinearityIQA) and the corresponding paper's Figure 5(c).
